src/app/theme/components/progressBarRound/progressBarRound.directive.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Created by n.poltoratsky
+ * on 28.06.2016.
+ */
+(function () {
+    'use strict';
+
+    angular.module('BlurAdmin.theme.components')
+        .directive('progressBarRound', progressBarRound);
+
+    /** @ngInject */
+    function progressBarRound(baProgressDialog) {
+        return {
+            restrict: 'E',
+            templateUrl: 'app/theme/components/progressBarRound/progressBarRound.html',
+            link:function($scope, element, attrs) {
+                $scope.baProgressDialog = baProgressDialog;
+                $scope.$watch('baProgressDialog.getProgress()', function() {
```

Comment:
If interpolation will not work for SVG attribute, write it like this:
```javascript
$scope.$watch(function() { 
  return baProgressDialog.getProgress();
}, animateBar);
```

first argument can be function as well